packages/azure/manifest.yml

```
@@ -25,12 +25,12 @@ screenshots:
 vars:
   - name: eventhub
     type: text
-    title: Event Hub
+    title: Event Hub Name
     multi: false
     required: true
     show_user: true
     description: >-
-      Elastic recommends using one event hub for each integration. Visit [Create an event hub](https://docs.elastic.co/integrations/azure#create-an-event-hub) to learn more. Use event hub names up to 30 characters long to avoid compatibility issues.
+      The event hub name that contains the logs to ingest. Do NOT use the event hub namespace here. Elastic recommends using one event hub for each integration. Visit [Create an event hub](https://docs.elastic.co/integrations/azure#create-an-event-hub) to learn more. Use event hub names up to 30 characters long to avoid compatibility issues.
```

```suggestion
      The event hub name that contains the logs to ingest. Do not use the event hub namespace here. Elastic recommends using one event hub for each integration. Visit [Create an event hub](https://docs.elastic.co/integrations/azure#create-an-event-hub) to learn more. Use event hub names up to 30 characters long to avoid compatibility issues.
```
Avoid capital letters in text.